pkg/kubelet/kubelet_network.go

```diff
@@ -95,10 +95,6 @@ func omitDuplicates(kl *Kubelet, pod *v1.Pod, combinedSearch []string) []string
 		if _, exists := uniqueDomains[dnsDomain]; !exists {
 			combinedSearch[len(uniqueDomains)] = dnsDomain
 			uniqueDomains[dnsDomain] = true
-		} else {
-			log := fmt.Sprintf("Found and omitted duplicated dns domain in host search line: '%s' during merging with cluster dns domains", dnsDomain)
-			kl.recorder.Event(pod, v1.EventTypeWarning, "DNSSearchForming", log)
-			glog.Error(log)
```

Comment:
@derekwaynecarr I thought it wasn't worthy of a log message at all.  It doesn't indicate an bad or undesired condition.  No point in clogging up the logs with something that is a normal operating condition.  In my mind, the only value this message could have is if there is a bug in the deduplication code that is erroneously removing a DNS search entry.